Tests/js/models/assets/ez-usermodel-tests.js

```diff
@@ -188,7 +191,107 @@ YUI.add('ez-usermodel-tests', function (Y) {
         },
     }));
 
+    loadDraftsTest = new Y.Test.Case({
+        name: "eZ User Model loadDrafts() method tests",
+
+        setUp: function () {
+            this.modelId = 'permanent-vacation';
+            this.model = new Y.eZ.User({id: this.modelId});
+        },
+
+        tearDown: function () {
+            this.model.destroy();
+
+            delete this.model;
+        },
+
+        "Should load user drafts": function () {
+            var model = this.model,
+                capiMock = new Y.Mock(),
+                isCallbackCalled = false,
+                contentServiceMock = new Y.Mock(),
+                draftHash = {
+                    "Version":{"_href":"/api/ezp/v2/content/objects/116/versions/3"},
+                    "VersionInfo":{
+                        "id":709,
+                        "versionNo":3,
+                        "status":"DRAFT",
+                        "modificationDate":"2016-06-03T14:20:49+02:00",
+                        "Creator":{"_href":"/api/ezp/v2/user/users/14"},
+                        "creationDate":"2016-06-03T14:20:49+02:00",
+                        "initialLanguageCode":"eng-GB",
+                        "languageCodes":"eng-GB",
+                        "names":{"value":[{"_languageCode":"eng-GB","#text":"2 zones"}]},
+                        "Content":{"_href":"/api/ezp/v2/content/objects/116"}
+                    }
+                },
+                loadUserDraftsResponse = {document: {VersionList: {VersionItem: [draftHash]}}},
+                loadDraftsCallback = function (error, response) {
+                    isCallbackCalled = true;
+
+                    Y.Assert.isFalse(error, 'Should not throw error');
+                    Y.Assert.areSame(1, response.length, 'Should return one draft model');
```

Comment:
instead of `1`, you should use `loadUserDraftsResponse.document.VersionList.VersionItem.length`